### PR TITLE
Add govuk-lint

### PR DESCRIPTION
**Pull request by tijmenb**
This should help us to maintain a cleaner code base.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -36,7 +35,6 @@
     t.string  "uid",                 limit: 255
     t.string  "organisation_slug",   limit: 255
     t.string  "permissions",         limit: 255
-    t.boolean "remotely_signed_out",             default: false
+    t.boolean "remotely_signed_out", default: false
   end
-
```
Comment (jamiecobbett) [inline db/schema.rb]: I think `schema.rb` should probably be excluded from linting as it's auto-generated.

Comment (tijmenb) [inline db/schema.rb]: Good point! Rebased and raised https://github.com/alphagov/govuk-lint/pull/41.

File: app/controllers/concerns/notifiable.rb [hunk 2]
```diff
@@ -7,7 +7,7 @@ def store_query_for_rummager(query)
   end
 
   def send_change_notification
-    attributes_to_send = @attributes_to_send.reject { |set| set.blank? }
+    attributes_to_send = @attributes_to_send.reject(&:blank?)
```
Comment (Davidslv) [inline app/controllers/concerns/notifiable.rb]: :+1:

**Comment by Davidslv** [comment]
looks good to me, going to merge

**Pull request merged**